### PR TITLE
[翻译完毕] extension_libraries

### PR DESCRIPTION
**Pull request by ArkhamWJZ**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/extension_libraries.po [hunk 1]
```diff
@@ -3,67 +3,71 @@
...
 
 #: ../../extension_libraries.rst:13
 msgid "Handy scalar functions like ``clamp``, ``smoothstep``, ``mix``, ``round``."
-msgstr ""
+msgstr "便捷的标量型函数，诸如 ``clamp``, ``smoothstep``, ``mix``, ``round``。"
 
 #: ../../extension_libraries.rst:14
 msgid "GLSL-alike vector functions like ``normalize``, ``distance``, ``reflect``."
-msgstr ""
+msgstr "类 GLSL 向量函数，诸如 ``normalize``, ``distance``, ``reflect``。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/extension_libraries.po]:
```suggestion
msgstr "类 GLSL 向量函数，诸如 ``normalize``，``distance``，``reflect``。"
```

File: locales/zh_CN/LC_MESSAGES/extension_libraries.po [hunk 2]
```diff
@@ -3,67 +3,71 @@
...
+"`Taichi GLSL <https://github.com/taichi-dev/taichi_glsl>`_ is an extension "
+"library of Taichi, aiming at providing useful helper functions including:"
 msgstr ""
+"`Taichi GLSL <https://github.com/taichi-dev/taichi_glsl>`_ 是一个Taichi 的扩展"
+"库， 其旨在提供各种有用的辅助函数，包括::"
 
 #: ../../extension_libraries.rst:13
 msgid "Handy scalar functions like ``clamp``, ``smoothstep``, ``mix``, ``round``."
-msgstr ""
+msgstr "便捷的标量型函数，诸如 ``clamp``, ``smoothstep``, ``mix``, ``round``。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/extension_libraries.po]:
```suggestion
msgstr "便捷的标量型函数，诸如 ``clamp``， ``smoothstep``，``mix``，``round``。"
```

File: locales/zh_CN/LC_MESSAGES/extension_libraries.po [hunk 3]
```diff
@@ -3,67 +3,71 @@
...
 
 #: ../../extension_libraries.rst:14
 msgid "GLSL-alike vector functions like ``normalize``, ``distance``, ``reflect``."
-msgstr ""
+msgstr "类 GLSL 向量函数，诸如 ``normalize``, ``distance``, ``reflect``。"
 
 #: ../../extension_libraries.rst:15
 msgid "Well-behaved random generators including ``randUnit2D``, ``randNDRange``."
-msgstr ""
+msgstr "表现良好的随机生成器，诸如 ``randUnit2D``, ``randNDRange``。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/extension_libraries.po]:
```suggestion
msgstr "表现良好的随机生成器，诸如 ``randUnit2D``，``randNDRange``。"
```

File: locales/zh_CN/LC_MESSAGES/extension_libraries.po [hunk 4]
```diff
@@ -3,67 +3,71 @@
...
 
 #: ../../extension_libraries.rst:16
 msgid "Handy vector and matrix initializer: ``vec`` and ``mat``."
-msgstr ""
+msgstr "便捷的向量和矩阵初始化方式，诸如 ``vec`` 和 ``mat``。"
 
 #: ../../extension_libraries.rst:17
 msgid "Handy vector component shuffle accessor like ``v.xy``."
-msgstr ""
+msgstr "便捷的向量分量随机访问，如 ``v.xy``。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/extension_libraries.po]:
```suggestion
msgstr "便捷的向量分量随机访问器，如 ``v.xy``。"
```
accessor? :thinking:

**Review by isdanni — COMMENTED** [review]
🎸 🚀 !

**Review by archibate — APPROVED** [review]
LGTM :+1:

**Pull request merged**